forge-gui/res/cardsfolder/upcoming/ohabi_caleria.txt

```diff
@@ -0,0 +1,9 @@
+Name:Ohabi Caleria
+ManaCost:1 G W
+Types:Legendary Creature Elf Archer
+PT:1/3
+K:Reach
+S:Mode$ Continuous | Affected$ Archer.YouCtrl | AddHiddenKeyword$ CARDNAME untaps during each other player's untap step. | Description$ Untap all Archers you control during each other player's untap step.
+T:Mode$ DamageDone | ValidSource$ Archer.YouCtrl | ValidTarget$ Creature | TriggerZones$ Battlefield | Execute$ DBDraw | TriggerDescription$ Whenever an Archer you control deals damage to a creature, you may pay {2}. If you do, draw a card.
+SVar:DBDraw:AB$ Draw | Cost$ 2 | NumCards$ 1
```

Comment:
This should be changed to SVar:DBDraw:DB$ Draw | Cost$ 2

Reply:
Your update only removed NumCards 1. The AB$ Draw needs to be changed to DB$ Draw.

Reply:
You sure ? if there's a cost involved it's ab I thought

Reply:
I was getting an error message when hovering over the card (that can be dismissed) that changing the parameter to DB$ Draw fixed. I'll try it again tomorrow with the latest snapshot.